src/background/background-init.ts

```diff
@@ -123,6 +124,9 @@ async function initialize(): Promise<void> {
 
     const telemetryEventHandler = new TelemetryEventHandler(telemetryClient);
 
+    const exceptionTelemetryListener = new ExceptionTelemetryListener(telemetryEventHandler);
```

Comment:
You mentioned in the PR description that only adding to "the background" was intentional - I wanted to clarify whether that meant "only the manifest v2 background page", or if this is intended to also be included in `service-worker-init`